src/components/Sidebar.tsx

```diff
@@ -0,0 +1,190 @@
+import React from 'react';
```

Comment:
This sidebar component looks great, but it's way too big right now. Can we split this into a Sidebar and a Header component separately?